roles/cache_builder/defaults/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+offline_cache_dir: "/tmp/deepops"
+build_tar_file: true
+tar_file_path: "/tmp/deepops-archive.tar.bz2"
+deepops_repo_url: "https://github.com/NVIDIA/deepops"
+
+centos_iso_mirror: "http://mirrors.ocf.berkeley.edu/centos/7.6.1810/isos/x86_64"
+centos_iso: "CentOS-7-x86_64-Minimal-1810.iso"
+download_yum_repos: yes
+yum_repos_to_sync:
+- "libnvidia-container"
+- "nvidia-container-runtime"
+- "nvidia-docker"
+- "cuda"
+- "base"
+- "updates"
+- "extras"
+- "centosplus"
+- "docker-ce-stable"
+- "docker-engine"
+- "epel"
+
+helm_download_url: "https://storage.googleapis.com/kubernetes-helm/helm-v2.11.0-linux-amd64.tar.gz"
+helm_chart_git_repos:
+  base: "https://github.com/helm/charts"
+rook_helm_chart_url: "https://charts.rook.io/master/rook-ceph-v0.9.0-79.g1a1ffdd.tgz"
+
+docker_image_list:
```

Comment:
The images that belong to kubespray should use the values found in kubespray's default role for the fixed version of kubespray you are installing.

Reply:
Good call. I'll check to see if there's a good way to hoist that variable into this role.

Reply:
Now including a run of the kubespray download role in `scripts/build_offline_cache.sh`.